bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -0,0 +1,362 @@
+Number:Temperature  	Biostar_BufferTop0             	"Buffer Top 0"                      	{ channel="guntamatic:biostar:mybiostar:bufferTop0" }
+Number:Temperature  	Biostar_BufferBtm0             	"Buffer Btm 0"                      	{ channel="guntamatic:biostar:mybiostar:bufferBtm0" }
+Number:Temperature  	Biostar_BufferTop1             	"Buffer Top 1"                      	{ channel="guntamatic:biostar:mybiostar:bufferTop1" }
+Number:Temperature  	Biostar_BufferBtm1             	"Buffer Btm 1"                      	{ channel="guntamatic:biostar:mybiostar:bufferBtm1" }
+Number:Temperature  	Biostar_BufferTop2             	"Buffer Top 2"                      	{ channel="guntamatic:biostar:mybiostar:bufferTop2" }
+Number:Temperature  	Biostar_BufferBtm2             	"Buffer Btm 2"                      	{ channel="guntamatic:biostar:mybiostar:bufferBtm2" }
+Switch              	Biostar_BExtraWw0              	"B extra-WW. 0"                     	{ channel="guntamatic:biostar:mybiostar:bExtraWw0" }
+Switch              	Biostar_BExtraWw1              	"B extra-WW. 1"                     	{ channel="guntamatic:biostar:mybiostar:bExtraWw1" }
+Switch              	Biostar_BExtraWw2              	"B extra-WW. 2"                     	{ channel="guntamatic:biostar:mybiostar:bExtraWw2" }
+Switch              	Biostar_AuxiliaryPump0         	"Auxiliary pump 0"                  	{ channel="guntamatic:biostar:mybiostar:auxiliaryPump0" }
+Switch              	Biostar_AuxiliaryPump1         	"Auxiliary pump 1"                  	{ channel="guntamatic:biostar:mybiostar:auxiliaryPump1" }
+Switch              	Biostar_AuxiliaryPump2         	"Auxiliary pump 2"                  	{ channel="guntamatic:biostar:mybiostar:auxiliaryPump2" }
+String              	Biostar_BoilersConditionNo     	"Boiler´s condition no."            	{ channel="guntamatic:biostar:mybiostar:boilersConditionNo" }
+Number:Temperature  	Biostar_BufferT5               	"Buffer T5"                         	{ channel="guntamatic:biostar:mybiostar:bufferT5" }
+Number:Temperature  	Biostar_BufferT6               	"Buffer T6"                         	{ channel="guntamatic:biostar:mybiostar:bufferT6" }
+Number:Temperature  	Biostar_BufferT7               	"Buffer T7"                         	{ channel="guntamatic:biostar:mybiostar:bufferT7" }
+Number:Temperature  	Biostar_ExtraWw0               	"Extra-WW. 0"                       	{ channel="guntamatic:biostar:mybiostar:extraWw0" }
+Number:Temperature  	Biostar_ExtraWw1               	"Extra-WW. 1"                       	{ channel="guntamatic:biostar:mybiostar:extraWw1" }
+Number:Temperature  	Biostar_ExtraWw2               	"Extra-WW. 2"                       	{ channel="guntamatic:biostar:mybiostar:extraWw2" }
+Number:Dimensionless	Biostar_Grate                  	"Grate"                             	{ channel="guntamatic:biostar:mybiostar:grate" }
+```
+
+**Rule**
+
+```javascript
```

Comment:
I believe you do not need to mention "javascript"

Reply:
> I believe you do not need to mention "javascript"

I've seen this in other README's.
With `javascript`, a proper syntax highlighting is used.
Without `javascript`, the text is rendered in black only.